docs/src/components/ui/Banner.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { styled } from "linaria/react";
-import { appTheme, tm } from "../../themes";
+import { appTheme, tm, tmDark, tmHCDark, tmSelectors } from "../../themes";
```

Comment:
we have two ways of importing a similar things. 
1 - here
2 in the `const { media, breakpoints } = appTheme;` line
can we consider switching to a simple single approach?